ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -41,6 +41,78 @@ EOF
     fi
 }
 
+# Update RHEL repo before upgrade
+function rhel_repo(){
+    echo "Updating RHEL node repo"
+    # Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
+    # to be able to SSH.
+    if ! whoami &> /dev/null; then
+        if [[ -w /etc/passwd ]]; then
+            echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+        else
+            echo "/etc/passwd is not writeable, and user matching this uid is not found."
+            exit 1
+        fi
+    fi
+    SOURCE_REPO_VERSION=$(echo "${SOURCE_VERSION}" | cut -d'.' -f1,2)
+    TARGET_REPO_VERSION=$(echo "${TARGET_VERSION}" | cut -d'.' -f1,2)
+    export SOURCE_REPO_VERSION
+    export TARGET_REPO_VERSION
+
+    cat > repo.yaml <<-'EOF'
+---
+- name: Update repo Playbook
+  hosts: new_workers
```

Comment:
More strictly speaking, it should be executed for [workers] group.

Reply:
```
[new_workers]
# hostnames must be listed by what `hostname -f` returns on the host
# this is the name the cluster will use
xxxx.compute.internal
yyyy.compute.internal
zzzz.compute.internal

[workers:children]
new_workers
```

Thanks! Let me update.